services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -991,11 +979,11 @@ async def is_hidden(self, project_uuid: ProjectID) -> bool:
             )
         return bool(result)
 
-    async def set_hidden_flag(self, project_uuid: ProjectID, enabled: bool):
+    async def set_hidden_flag(self, project_uuid: ProjectID, *, hidden: bool):
```

Comment:
For my Python-knowledge: What is the purpose of adding `*` as an argument here?

Reply:
It forces the caller of that function to use keyword argument for ```hidden``` argument. Also we use the Ruff linter that uses flake. this is one of them good practices.

Reply:
cool, thanks